code/modules/mob/living/simple_animal/hostile/megafauna/legion.dm

```diff
@@ -57,8 +57,6 @@
 				charging = 0
 
 /mob/living/simple_animal/hostile/megafauna/legion/death()
-	if(health > 0)
```

Comment:
Why? If you wand of death Legion, it just splits in two, and you no longer have your WoD.